src/NUnitEngine/nunit-agent/nunit-agent-x86.csproj

```diff
@@ -2,16 +2,20 @@
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <RootNamespace>nunit.agent</RootNamespace>
-    <TargetFrameworks>net20</TargetFrameworks>
+    <TargetFrameworks>net20;net40</TargetFrameworks>
     <ApplicationManifest>app.manifest</ApplicationManifest>
     <ApplicationIcon>..\..\..\nunit.ico</ApplicationIcon>
     <PlatformTarget>x86</PlatformTarget>
     <IntermediateOutputPath>obj\$(Configuration)\x86\</IntermediateOutputPath>
+    <OutputPath>..\..\..\bin\$(Configuration)\agents\</OutputPath>
     <GenerateSupportedRuntime>false</GenerateSupportedRuntime>
   </PropertyGroup>
   <ItemGroup Condition="'$(TargetFramework)'=='net20'">
     <Reference Include="System.Runtime.Remoting" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetFramework)'=='net40'">
```

Comment:
I would prefer to have an or in the condition rather that two separate ItemGroups unless we expect that the references are going to be different on the two targets.

Reply:
It's interesting that with the new SDK-style projects, readability of project files becomes a thing, just as it was once a thing for Nant scripts in the distant past.

Of course, the question is "What do we think is most readable here?" I thought separating each one would be better, although it's not a very strong preference for me.

@ChrisMaddock @jnm2 What do you think? What's the best practice for combining or not combining these in your views.

One suggestion. Let's avoid __negative__ tests. The problem with them is that they impact runtimes that are added __later on__.

Reply:
I'll leave this as is until there's some more feedback and change as needed in a later commit.

Reply:
I would expect an 'or' by convention, but don't feel strongly. I guess it would highlight the similarities of the two platform builds to a future contributor.

> readability of project files becomes a thing

One of the biggest steps forward in my time in .NET, in my opinion! 😄 The less cryptic magic buried in enormous XML files, the better. If anything, I wish the VS experience was a little more oriented towards editing the xml directly, rather than still be GUI-first.

Reply:
OK... easy fix.

Reply:
Ah! Not quite so easy because VS wants to create the runtime subdirectories. I guess that's why I made a parallel agents subdirectory (it's been a few weeks, I'm afraid).

Reply:
VS should not behave differently at all when reading this versus the original:

```cs
  <ItemGroup Condition="'$(TargetFramework)'=='net20' or '$(TargetFramework)'=='net40'">
    <Reference Include="System.Runtime.Remoting" />
  </ItemGroup>
```

Reply:
Correct. But people have stylistic preferences, so I'll change it. Of course, the original didn't have net40 at all.